client/blocks/announcement-modal/docs/example.jsx

```diff
@@ -1,20 +1,55 @@
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PlaceholderImage from 'calypso/assets/images/marketplace/plugins-revamp.png';
 import AnnouncementModal from 'calypso/blocks/announcement-modal';
-import PlaceholderImage from 'calypso/images/extensions/woocommerce/image-placeholder.png';
+import Button from 'calypso/components/forms/form-button';
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { hasReceivedRemotePreferences } from 'calypso/state/preferences/selectors';
+
+const AnnouncementModalExample = () => {
+	const announcementId = 'announcement-modal-example';
```

Comment:
Nit: the preference name already contains `announcement-modal`, so do we want to write it again as part of the announcement ID?

Reply:
Good catch!

Reply:
Changing it to `const announcementId = 'example'`